app/javascript/components/User/Scan/states.json

```diff
@@ -0,0 +1,6 @@
+{
+  "SCANNING": "SCANNING",
+  "ERROR_QR_READ": "ERROR_QR_READ",
+  "ERROR_MERCHAND": "ERROR_MERCHAND",
+  "SUCCESS": "SUCCESS"
+}
```

Comment:
Since this is not holding any data, shouldn't this just be a `.js` file as others?